packages/cloudflare/src/cli/args.ts

```diff
@@ -81,9 +84,27 @@ export function getArgs(): Arguments {
   }
 }
 
-function getPassthroughArgs() {
-  const passthroughPos = process.argv.indexOf("--");
-  return passthroughPos === -1 ? [] : process.argv.slice(passthroughPos + 1);
+export function getPassthroughArgs<T extends ParseArgsConfig>(args: string[], { options = {} }: T) {
+  const passthroughArgs: string[] = [];
+
+  args.forEach((fullArg, idx) => {
+    const [, name] = /^--?(\w[\w-_]*)(=.+)?$/.exec(fullArg) ?? [];
+    if (name && !(name in options)) {
+      passthroughArgs.push(fullArg);
+
+      for (let i = idx + 1; i < args.length; i++) {
```

Comment:
not sure to understand why we need this nested loop?

Reply:
This was to append strings that follow the flag